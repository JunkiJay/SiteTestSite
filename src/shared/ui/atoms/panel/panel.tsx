import { styled } from '@mui/material/styles';
import { Paper } from '@mui/material';

export const Panel = styled(Paper)`
  border-radius: 10px;
  padding: 10px;
`;
