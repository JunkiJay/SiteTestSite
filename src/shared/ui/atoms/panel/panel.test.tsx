import { render, screen } from '@testing-library/react';
import { Panel } from './panel';

describe('Panel', () => {
  it('Match Snapshot', () => {
    const { container } = render(<Panel>Test</Panel>);

    expect(screen.getByText('Test')).toHaveTextContent('Test');

    expect(container).toMatchSnapshot();
  });

  it('Rendered', () => {
    render(<Panel>Test</Panel>);

    const element = screen.getByText('Test');

    expect(element).toBeInTheDocument();
  });
});
