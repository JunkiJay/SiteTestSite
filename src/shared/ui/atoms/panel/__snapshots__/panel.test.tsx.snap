// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Panel Match Snapshot 1`] = `
<div>
  <div
    class="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation1 css-8cur7s-MuiPaper-root"
  >
    Test
  </div>
</div>
`;
